benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -30,30 +30,31 @@ def get_name(rank, configs):
         return "master"
 
 
-def get_parameter_server_rank(rank, config):
+def get_ps_rank(rank, config):
     # rank mod parameter server count to get parameter server number
     # add trainer_count to get parameter server rank
     rank_mod_ps_count = rank % config.ps_count
     return rank_mod_ps_count + config.trainer_count
 
 
-def get_ps_rref(parameter_server_rank, config):
+def get_ps_rref(ps_rank, config):
     ps_config = config.ps_config
     ps = get_benchmark_ps_map()[str(ps_config["ps_class"])]
     name = get_name(
-        parameter_server_rank,
+        ps_rank,
         config
     )
     ps_args = ps_config["configurations"].values()
-    ps_trainer_count = config.trainer_count / ps_config.ps_count
-    rem = config.trainer_count % ps_config.ps_count
-    if parameter_server_rank - config.trainer_count < rem:
+    ps_trainer_count = config.trainer_count / config.ps_count
+    rem = config.trainer_count % config.ps_count
+    if ps_rank - config.trainer_count < rem:
         ps_trainer_count += 1
+    ps_trainer_count = int(ps_trainer_count)
```

Comment:
nit: you can just make line 48 `ps_trainer_count = config.trainer_count // config.ps_count` to accomplish the same thing

Question, do we have any restrictions on the # of parameter server and trainer counts?

Reply:
no restriction on server and trainer counts. this benchmark is set to run within a single node. the dev queue has 4 gpus per node. train queue has 8 gpus per node. 

I have the benchmark set so that the number of trainers must be divisible by the number of servers.